crates/turbo-tasks-hash/src/deterministic_hash.rs

```diff
@@ -128,6 +128,15 @@ impl<T: DeterministicHash> DeterministicHash for Option<T> {
     }
 }
 
+impl<T: DeterministicHash> DeterministicHash for Vec<T> {
+    fn deterministic_hash<H: DeterministicHasher>(&self, state: &mut H) {
+        state.write_usize(self.len());
```

Comment:
Is this necessary? I expect hashing all items in order will already produce a deterministic hash for a vector.

Reply:
Very nice catch. TIL!